packages/flutter/test/foundation/_with_flutter_leak_tracking.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:leak_tracker/leak_tracker.dart';
+
+/// Wrapper for [withLeakTracking] with Flutter specific functionality.
```

Comment:
The doc comment could be a little more comprehensive. Questions I have after reading this that I wish would be answered:
* What does this do on a high-level? What's the benefit of using this?
* How am I expected to use this function? 
* How do I make sure there are no leaks? 
* What happens if there are leaks detected?
* What is the Flutter specific functionality mentioned in the first sentence?
* What do all the method parameters mean? What should I pass in?

Reply:
Linked documentation in leak_tracker that partially covers the questions.
Can you approve: https://github.com/dart-lang/leak_tracker/pull/39